scripts/publish-to-unity-package.bat

```diff
@@ -19,9 +19,25 @@ echo Moving files to Unity package...
 pushd bin\release\netstandard2.1\publish
 del Newtonsoft.Json.dll
 del UnityEngine.dll
-if not exist ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries mkdir ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\
-del ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\* /F /Q
-copy *.dll ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
+
+REM Check if io.chainsafe.web3-unity.lootboxes directory exists
+if exist ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity.lootboxes (
```

Comment:
Question to think about: Is this the only Lootboxes implementation we're going to have? If not, then we better come up with a more specific name in my opinion.

Also, I've consciously added this `Chainlink()` node in case we decide to add another Lootboxes implementation. Should we remove it then?

![image](https://github.com/ChainSafe/web3.unity/assets/127766834/32a51e5f-0954-4116-991f-f663e73720fb)

@kalambet